azure-template-deployment/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+az group create --name "$RESOURCE_GROUP_NAME" --location "$RESOURCE_GROUP_LOCATION"
```

Comment:
Will this not fail, if the RG already exists?

Reply:
no it is a no op if RG exists

Reply:
i have added check for RG creation